ams/server/src/main/java/com/netease/arctic/server/dashboard/model/SqlExample.java

```diff
@@ -36,10 +36,13 @@ public enum SqlExample {
       "alter table db_name.table_name add column data int ;\n"
           + "alter table db_name.table_name alter column data bigint ;\n"
           + "alter table db_name.table_name drop column data;"),
-  ALTER_PROPERTIES(
-      "AlterProperties",
+  ADD_PROPERTIES(
+      "AddProperties",
```

Comment:
Is `SetProperties` proper here? the SQL can both add and update properties.

Reply:
Yeah, more reasonable.